vip/test/.test_brito.py

```diff
@@ -0,0 +1,165 @@
+import bito
```

Comment:
I'm guessing that this file and the one below are cruft? Don't forget about using .gitignore or the _ignore directory.